src/sage/symbolic/expression_conversion_algebraic.py

```diff
@@ -97,6 +98,28 @@ def arithmetic(self, ex, operator):
             sage: L = QuadraticField(3, embedding=-AA(3).sqrt())
             sage: bool(L.gen() == -sqrt(3))
             True
+
+        Test that deprecation warnings are issued correctly::
+
+            sage: AA((-1)^(2/3))
+            doctest:warning...
+            DeprecationWarning: Conversion of negative number to fractional power to AA is deprecated
+            See https://github.com/sagemath/sage/issues/36735 for details.
+            1
+            sage: AA((-sqrt(27)+1)^5)
+            -1300.937032397459?
+            sage: AA((-sqrt(27)+1)^(5/2))
+            Traceback (most recent call last):
+            ...
+            ValueError: Cannot coerce algebraic number with nonzero imaginary part to algebraic real
+
+        Another test (need to be splitted from the above for the warning to appear)::
+
+            sage: AA((-sqrt(27)+1)^(5/3))
```

Comment:
Instead of "Test that deprecation warnings are issued correctly", how about explaining what is deprecated? As this is for users, perhaps we may place these examples in "EXAMPLES" block.